frontend/src/components/search/SearchResultsListItem.tsx

```diff
@@ -75,12 +75,12 @@ export default function SearchResultsListItem({
                   : "--"}
               </span>
               {saved && (
-                <span className="padding-x-105 padding-y-1px bg-base-lighter display-flex flex-align-center font-sans-2xs">
-                    <USWDSIcon
-                      name="star"
-                      className="text-accent-warm-dark button-icon-large padding-right-05"
-                    />
-                    {t("opportunitySaved")}
+                <span className="padding-x-105 padding-y-2px bg-base-lighter display-flex flex-align-center font-sans-3xs radius-sm">
```

Comment:
Nit: All the size/padding/radius classes could probably be replaced with `.usa-tag`

Reply:
@acouch `.usa-tag` applies a text-transform of uppercase. USWDS doesn't have a `text-transform: capitalize` so it would need to be a custom tag:

![image](https://github.com/user-attachments/assets/24ca086b-a35c-45c0-bccc-9a8741dfbc0f) 

Additionally the padding is different because the font size of the "Saved" is different.

So it is `usa-tag simpler-text-capitalize color-black` plus the same padding and font-size additions.

Reply:
Noting in Figma this is a "tag2". I assume that this started from the USWDS Figma file. Might be helpful to note any of the variants ahead of time to address how they can be addressed

<img width="872" alt="image" src="https://github.com/user-attachments/assets/68a0e5b3-f599-4f22-bc3a-97a157cd12ee" />

Reply:
D'oh! I see. Yeah, we don't want these ALL CAPS YELLING